src/oop/programs_with_multiple_classes/hard/q2/MainTest.java

```diff
@@ -0,0 +1,191 @@
+package oop.programs_with_multiple_classes.hard.q2;
+
+import global.BaseTest;
+import global.ObjectTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest extends BaseTest {
+
+    private ObjectTest person;
+    private ObjectTest blanket;
+
+    private final String height = "height";
```

Comment:
I think for these fields which describe the names of class fields, instead of naming them by whatever the original name was, we should name them like "personStringFieldName"